demisto_sdk/tests/test_files/content_repo_example/Packs/FeedAzure/TestPlaybooks/just_a_test_script.yml

```diff
@@ -15,3 +15,5 @@ args:
   description: Amount of seconds to sleep
 scripttarget: 0
 timeout: 1h40m0s
```

Comment:
Why are you using all these test files in this `FeedAzure` test pack?

Reply:
Other tests issues